docs/source/metrics.rst

```diff
@@ -39,6 +39,13 @@ Metrics
 .. autoclass:: DiceMetric
     :members:
 
+`Mean IoU`
+-----------
```

Comment:
Please ensure the length of `---` matches the string.

Thanks.

Reply:
Done!